src/styles/headerStyles.scss

```diff
@@ -0,0 +1,64 @@
+@import "./value.scss";
```

Comment:
no need to import it in every file - import it once in main.scss in the first line
the same with other similar imports